From felipe.balbi@nokia.com  Fri Apr 23 14:05:40 2010
From: Sergei Shtylyov <sshtylyov@ru.mvista.com>
Date: Thu, 25 Mar 2010 13:14:31 +0200
Subject: MUSB: OMAP: don't call clk_put()
To: Greg KH <greg@kroah.com>
Message-ID: <1269515673-27980-10-git-send-email-felipe.balbi@nokia.com>


From: Sergei Shtylyov <sshtylyov@ru.mvista.com>

Remove duplicate/unbalanced call to clk_put() from musb_platform_exit() --
clk_put() gets called from musb_core.c anyway...

Signed-off-by: Sergei Shtylyov <sshtylyov@ru.mvista.com>
Signed-off-by: Felipe Balbi <felipe.balbi@nokia.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/musb/omap2430.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/drivers/usb/musb/omap2430.c
+++ b/drivers/usb/musb/omap2430.c
@@ -331,8 +331,5 @@ int musb_platform_exit(struct musb *musb
 
 	musb_platform_suspend(musb);
 
-	clk_put(musb->clock);
-	musb->clock = NULL;
-
 	return 0;
 }
